### PR TITLE
Adding a file for London, UK

### LINKED ISSUE #28: Hometown for jsaonlineuk
## Add file for London, UK

- [x]  Create branch
- [x]  Add a file for London
- [x]  Commit my file
- [x]  Open a pull request
- [ ]  Have a discussion
- [ ]  Merge my pull request

### PR DESCRIPTION
**Pull request by jsaonlineuk**
This PR closes #28 @douglasreis  :+1:

### CODE REVIEW

File: places/london-uk.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## London, UK
+
+### Places to go
+- Savoy Grill
+- Meat Licqour
+- Dinner (Heston)
```
Comment (jsaonlineuk) [inline places/london-uk.md]: What is up with this?

Comment (douglasreis) [inline places/london-uk.md]: London?!?!?!

Comment (jsaonlineuk) [inline places/london-uk.md]: Well, to make it easier. Do you know how long it takes to write Pietermaritzburg?!

**Comment by HarryMiddelburg** [comment]
Hello London
where is London Bridge?

**Comment by douglasreis** [comment]
Finally!!! kkkkk

**Comment by jsaonlineuk** [comment]
Haha!

**Pull request merged**